CHANGES.txt

```diff
@@ -1,4 +1,4 @@
-﻿Notable changes in recent builds.
+Notable changes in recent builds.
```

Comment:
what's this phantom change?

Reply:
I think `CHANGES.txt` has a weird hidden first character (sometimes it shows up as a red dot in GitHub, I've been noticing it for a while).

I guess when I discarded this file form GitHub it must've changed that. I'll try again using the CLI.

Reply:
I think it's a Byte Order Mark. VSCode opens this file as `UTF-8 with BOM` by default.